### DISCUSSION TITLE
Authentik bypassed by apps ??

### DISCUSSION THREAD

**Discussion by ThierryIT** [goauthentik/authentik#9173] [Q&A]
Hello, I have installed Authentik to protect the apps accessible from internet ... 
This morning I have try to access uptime-kuma directly from my phone, and it work without any authentification ..... Any ideas ?

the provider config
```
default-source-authentication (Welcome to authentik!)
default-provider-authorization-explicit-consent (Authorize Application)
mode proxy
```

**Comment by ThierryIT** [comment]
nobody ?

**Comment by Keyinator** [comment]
Are you trying to use authentik as a proxy or forward-auth?
